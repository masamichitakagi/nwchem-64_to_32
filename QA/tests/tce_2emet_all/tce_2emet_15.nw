echo

start tce_2emet_15

memory stack 900 mb heap 100 mb global 900 mb

#permanent_dir /tmp
#scratch_dir /tmp

geometry units angstrom noprint
 # Aces CCSD(T)/aug-cc-pVQZ(ae) optimized geometry
 zmatrix
  O
  H 1 R
  H 1 R 2 A
  variables
   R   0.9575091749
   A 104.4784445984
  end
end

basis spherical 
  * library cc-pVDZ
end

scf
  singlet
  rhf
  thresh 1e-9
#  semidirect memsize 50000000 filesize 0
end

tce
  freeze atomic
  scf
  ccsd
  thresh 1e-7
  io ga
  2eorb
  2emet 15
  tilesize 20
  attilesize 30
end

task tce energy

