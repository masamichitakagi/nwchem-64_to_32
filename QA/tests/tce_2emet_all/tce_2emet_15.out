refund: UserID = kowalski
refund: SLURM Job ID = 6683580
refund: Number of nodes          = 1
refund: Number of cores per node = 16
refund: Number of cores          = 16
refund: Amount of time requested = 0:20
refund: Directory = /people/kowalski/nwchem_6_5_branch/QA/tests/tce_2emet_all
 
Processor list
 
node0117
 
/share/apps/intel/11.1/mkl/lib/em64t:/usr/local/mvapich2/1.8/gcc/4.6.2/lib:/share/apps/gcc/4.6.2/lib:/share/apps/gcc/4.6.2/lib64:/usr/local/lib
 argument  1 = /people/kowalski/nwchem_6_5_branch/QA/tests/tce_2emet_all/tce_2emet_15.nw
 


============================== echo of input deck ==============================
echo

start tce_2emet_15

memory stack 900 mb heap 100 mb global 900 mb

#permanent_dir /tmp
#scratch_dir /tmp

geometry units angstrom noprint
 # Aces CCSD(T)/aug-cc-pVQZ(ae) optimized geometry
 zmatrix
  O
  H 1 R
  H 1 R 2 A
  variables
   R   0.9575091749
   A 104.4784445984
  end
end

basis spherical
  * library cc-pVDZ
end

scf
  singlet
  rhf
  thresh 1e-9
#  semidirect memsize 50000000 filesize 0
end

tce
  freeze atomic
  scf
  ccsd
  thresh 1e-7
  io ga
  2eorb
  2emet 15
  tilesize 20
  attilesize 30
end

task tce energy

================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.3
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = node0117.local
    program         = /scratch/nwchem
    date            = Wed Aug 27 11:39:36 2014

    compiled        = Tue_Aug_26_16:01:49_2014
    source          = /pic/people/kowalski/nwchem_6_5_branch
    nwchem branch   = Development
    nwchem revision = 26056
    ga revision     = 10502
    input           = /people/kowalski/nwchem_6_5_branch/QA/tests/tce_2emet_all/tce_2emet_15.nw
    prefix          = tce_2emet_15.
    data base       = ./tce_2emet_15.db
    status          = startup
    nproc           =       16
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107194 doubles =    100.0 Mbytes
    stack    =  117964799 doubles =    900.0 Mbytes
    global   =  117964800 doubles =    900.0 Mbytes (distinct from heap & stack)
    total    =  249036793 doubles =   1900.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
 C2V symmetry detected
  warning:::::::::::::: from_environment
  NWCHEM_BASIS_LIBRARY set to: <~scicons/apps/nwchem-6.3//src/basis/libraries/>
  but file does not exist !
  using .nwchemrc or compiled library


 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pVDZ                   on all atoms 


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.17200000E+04  0.000710
  1 S  1.75900000E+03  0.005470
  1 S  4.00800000E+02  0.027837
  1 S  1.13700000E+02  0.104800
  1 S  3.70300000E+01  0.283062
  1 S  1.32700000E+01  0.448719
  1 S  5.02500000E+00  0.270952
  1 S  1.01300000E+00  0.015458
 
  2 S  1.17200000E+04 -0.000160
  2 S  1.75900000E+03 -0.001263
  2 S  4.00800000E+02 -0.006267
  2 S  1.13700000E+02 -0.025716
  2 S  3.70300000E+01 -0.070924
  2 S  1.32700000E+01 -0.165411
  2 S  5.02500000E+00 -0.116955
  2 S  1.01300000E+00  0.557368
 
  3 S  3.02300000E-01  1.000000
 
  4 P  1.77000000E+01  0.043018
  4 P  3.85400000E+00  0.228913
  4 P  1.04600000E+00  0.508728
 
  5 P  2.75300000E-01  1.000000
 
  6 D  1.18500000E+00  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30100000E+01  0.019685
  1 S  1.96200000E+00  0.137977
  1 S  4.44600000E-01  0.478148
 
  2 S  1.22000000E-01  1.000000
 
  3 P  7.27000000E-01  1.000000
 


 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pVDZ                  6       14   3s2p1d
 H                          cc-pVDZ                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------
 
 

  ao basis        = "ao basis"
  functions       =    24
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./tce_2emet_15.movecs
  use symmetry    = T
  symmetry adapt  = T


 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pVDZ                  6       14   3s2p1d
 H                          cc-pVDZ                  3        5   2s1p


      Symmetry analysis of basis
      --------------------------
 
        a1         11
        a2          2
        b1          7
        b2          4
 

 Forming initial guess at       0.1s

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.76222910
 
      Non-variational initial energy
      ------------------------------

 Total energy =     -75.925641
 1-e energy   =    -121.769702
 2-e energy   =      36.651967
 HOMO         =      -0.469360
 LUMO         =       0.091304
 
 
      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------
 
  Numbering of irreducible representations: 
 
     1 a1          2 a2          3 b1          4 b2      
 
  Orbital symmetries:
 
     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1          8 b1          9 a1         10 a1      
    11 b2         12 b1         13 a1         14 a2         15 b2      
 

 Starting SCF solution at       0.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-09
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-11
 ----------------------------------------------


 #quartets = 1.953D+03 #integrals = 1.482D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_2emet_15.aoints.00
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 108878
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -75.9918898393  8.32D-01  3.67D-01      0.2
                 2      -76.0245068898  1.73D-01  7.81D-02      0.2
                 3      -76.0267685934  1.47D-02  6.37D-03      0.2
                 4      -76.0267848611  3.42D-05  1.90D-05      0.2
                 5      -76.0267848612  2.15D-10  1.22D-10      0.2


       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.026784861212
      One-electron energy =   -123.145951537815
      Two-electron energy =     37.927072037845
 Nuclear repulsion energy =      9.192094638758

        Time for solution =      0.1s


 
       Symmetry analysis of molecular orbitals - final
       -----------------------------------------------
 
  Numbering of irreducible representations: 
 
     1 a1          2 a2          3 b1          4 b2      
 
  Orbital symmetries:
 
     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1          8 b1          9 a1         10 a1      
    11 b2         12 b1         13 a1         14 a2         15 b2      
 
             Final eigenvalues
             -----------------

              1      
    1  -20.5505
    2   -1.3366
    3   -0.6991
    4   -0.5666
    5   -0.4931
    6    0.1855
    7    0.2562
    8    0.7890
    9    0.8542
   10    1.1636
   11    1.2004
   12    1.2534
   13    1.4443
   14    1.4763
   15    1.6744
 
                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.336584D+00  Symmetry=a1
              MO Center=  5.0D-16, -1.0D-16,  5.4D-02, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.442904  1 O  s                  3      0.375854  1 O  s          
    15      0.193522  2 H  s                 20      0.193522  3 H  s          
 
 Vector    3  Occ=2.000000D+00  E=-6.990915D-01  Symmetry=b1
              MO Center= -3.2D-16, -8.3D-18,  1.1D-01, r^2= 7.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.489963  1 O  px                15     -0.327925  2 H  s          
    20      0.327925  3 H  s                  7      0.221991  1 O  px         
 
 Vector    4  Occ=2.000000D+00  E=-5.665957D-01  Symmetry=a1
              MO Center=  7.5D-18, -1.4D-33, -1.6D-01, r^2= 6.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.545426  1 O  pz                 9      0.365270  1 O  pz         
     3     -0.349831  1 O  s                 15      0.206439  2 H  s          
    20      0.206439  3 H  s                  2     -0.150419  1 O  s          
 
 Vector    5  Occ=2.000000D+00  E=-4.931403D-01  Symmetry=b2
              MO Center=  1.5D-17,  2.3D-16, -9.3D-02, r^2= 6.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.631182  1 O  py                 8      0.495657  1 O  py         
 
 Vector    6  Occ=0.000000D+00  E= 1.855187D-01  Symmetry=a1
              MO Center= -6.4D-17, -2.5D-16,  6.1D-01, r^2= 3.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.002469  1 O  s                 16     -0.829188  2 H  s          
    21     -0.829188  3 H  s                  9      0.337123  1 O  pz         
     6      0.190609  1 O  pz         
 
 Vector    7  Occ=0.000000D+00  E= 2.562187D-01  Symmetry=b1
              MO Center=  1.0D-16, -2.1D-32,  6.2D-01, r^2= 3.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      1.444179  2 H  s                 21     -1.444179  3 H  s          
     7      0.670957  1 O  px                 4      0.283244  1 O  px         
 
 Vector    8  Occ=0.000000D+00  E= 7.890111D-01  Symmetry=b1
              MO Center=  5.0D-15, -4.2D-17,  2.5D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.944338  2 H  s                 20     -0.944338  3 H  s          
    16     -0.686518  2 H  s                 21      0.686518  3 H  s          
     7      0.460933  1 O  px                 4      0.267905  1 O  px         
    19      0.152786  2 H  pz                24     -0.152786  3 H  pz         
 
 Vector    9  Occ=0.000000D+00  E= 8.542440D-01  Symmetry=a1
              MO Center= -5.6D-15,  5.6D-17,  4.7D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.787436  2 H  s                 20      0.787436  3 H  s          
    16     -0.547871  2 H  s                 21     -0.547871  3 H  s          
     6     -0.329479  1 O  pz                 3      0.320257  1 O  s          
    17     -0.295994  2 H  px                22      0.295994  3 H  px         
     2     -0.255966  1 O  s          
 
 Vector   10  Occ=0.000000D+00  E= 1.163552D+00  Symmetry=a1
              MO Center=  1.6D-17,  3.9D-17, -1.4D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.279088  1 O  pz                 6     -0.754320  1 O  pz         
     3      0.748544  1 O  s                 15     -0.546923  2 H  s          
    20     -0.546923  3 H  s                 19      0.250920  2 H  pz         
    24      0.250920  3 H  pz         
 
 Vector   11  Occ=0.000000D+00  E= 1.200374D+00  Symmetry=b2
              MO Center=  9.0D-17,  1.0D-16, -1.1D-01, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8     -1.025047  1 O  py                 5      0.967823  1 O  py         
 
 Vector   12  Occ=0.000000D+00  E= 1.253353D+00  Symmetry=b1
              MO Center=  4.9D-16, -3.0D-33, -1.2D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.763959  1 O  px                16      0.825277  2 H  s          
    21     -0.825277  3 H  s                  4     -0.734251  1 O  px         
    15      0.379205  2 H  s                 20     -0.379205  3 H  s          
    17      0.302575  2 H  px                22      0.302575  3 H  px         
    19     -0.186806  2 H  pz                24      0.186806  3 H  pz         
 
 Vector   13  Occ=0.000000D+00  E= 1.444322D+00  Symmetry=a1
              MO Center= -3.5D-16,  7.7D-19,  5.9D-02, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.740230  1 O  pz                19     -0.545309  2 H  pz         
    24     -0.545309  3 H  pz                 2      0.528611  1 O  s          
     3     -0.505216  1 O  s                 15     -0.333692  2 H  s          
    20     -0.333692  3 H  s                 17     -0.329718  2 H  px         
    22      0.329718  3 H  px                16      0.209312  2 H  s          
 
 Vector   14  Occ=0.000000D+00  E= 1.476307D+00  Symmetry=a2
              MO Center=  6.2D-15,  2.2D-17,  4.3D-01, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      0.685618  2 H  py                23     -0.685618  3 H  py         
 
 Vector   15  Occ=0.000000D+00  E= 1.674437D+00  Symmetry=b2
              MO Center= -7.4D-15, -6.3D-17,  2.9D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      0.767025  2 H  py                23      0.767025  3 H  py         
     8     -0.633734  1 O  py                11      0.160785  1 O  d -1       
 

 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =  -0.09760123

 moments of inertia (a.u.)
 ------------------
           2.197735433895           0.000000000000           0.000000000000
           0.000000000000           6.322357628831           0.000000000000
           0.000000000000           0.000000000000           4.124622194936
 
  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     8.31   2.00  0.83  0.82  2.82  1.81  0.01
    2 H    1     0.85   0.69  0.07  0.09
    3 H    1     0.85   0.69  0.07  0.09
 
       Multipole analysis of the density wrt the origin
       ------------------------------------------------
 
     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      0.000000      0.000000     10.000000
 
     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.809336      0.000000      0.000000
 
     2   2 0 0     -3.065051      0.000000      4.092598
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -5.229500      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -4.375338      0.000000      1.964381
 

 Parallel integral file used      16 records with       0 large values

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------
 
              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================
 
            General Information
            -------------------
      Number of processors :    16
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
           No. of orbitals :    48
            Alpha orbitals :    24
             Beta orbitals :    24
        Alpha frozen cores :     1
         Beta frozen cores :     1
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    24
       Number of AO shells :    12
        Use of symmetry is : on 
      Symmetry adaption is : on 
         Schwarz screening : 0.10D-09
 
          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library
        L-threshold : 0.10D-06
        EOM-threshold : 0.10D-06
 no EOMCCSD initial starts read in
 TCE RESTART OPTIONS
 READ_INT:    F
 WRITE_INT:   F
 READ_TA:     F
 WRITE_TA:    F
 READ_XA:     F
 WRITE_XA:    F
 READ_IN3:    F
 WRITE_IN3:   F
 SLICE:       F
 D4D5:        F
 
            Memory Information
            ------------------
          Available GA space size is    1887436224 doubles
          Available MA space size is     131070188 doubles
 
 Maximum block size supplied by input
 Maximum block size        20 doubles

 tile_dim =      8

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a1     2 doubles       0       1
   2    alpha     b1     1 doubles       2       2
   3    alpha     b2     1 doubles       3       3
   4    beta      a1     2 doubles       4       1
   5    beta      b1     1 doubles       6       2
   6    beta      b2     1 doubles       7       3
   7    alpha     a1     8 doubles       8       7
   8    alpha     a2     2 doubles      16       8
   9    alpha     b1     6 doubles      18       9
  10    alpha     b2     3 doubles      24      10
  11    beta      a1     8 doubles      27       7
  12    beta      a2     2 doubles      35       8
  13    beta      b1     6 doubles      37       9
  14    beta      b2     3 doubles      43      10

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 #quartets = 3.081D+03 #integrals = 2.434D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_2emet_15.aoints.00
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 108878
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0

 
 Fock matrix recomputed
 1-e file size   =              169
 1-e file name   = ./tce_2emet_15.f1   
 Cpu & wall time / sec            0.0            0.1
 4-electron integrals stored in orbital form
 
 v2    file size   =            20937
 4-index algorithm nr.  15 is used
 imaxsize =       30
 imaxsize ichop =        0
 starting step 0 at                 0.43 secs 
 starting step 1 at                 0.84 secs 
 starting step 2 at                 0.86 secs 
 starting step 3 at                 0.86 secs 
 starting step 4 at                 0.87 secs 
 done step 4 at                 0.87 secs 
 Cpu & wall time / sec            0.4            0.4
 do_pt =   F
 do_lam_pt =   F
 do_cr_pt =   F
 do_lcr_pt =   F
 do_2t_pt =   F
 T1-number-of-tasks                        3
 
 t1 file size   =               25
 t1 file name   = ./tce_2emet_15.t1   
 t1 file handle =       -998
 T2-number-of-boxes                       54
 
 t2 file size   =             2426
 t2 file name   = ./tce_2emet_15.t2   
 t2 file handle =       -996

 CCSD iterations
 -----------------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall    V2*C2
 -----------------------------------------------------------------
    1   0.1055060040651  -0.2016397398372     0.0     0.0     0.0
    2   0.0295458973952  -0.2073949296824     0.0     0.0     0.0
    3   0.0105955716635  -0.2101186528927     0.0     0.0     0.0
    4   0.0042174675417  -0.2107977957947     0.0     0.0     0.0
    5   0.0017533470749  -0.2110541803321     0.0     0.0     0.0
 MICROCYCLE DIIS UPDATE:                        5                        5
    6   0.0001813343451  -0.2112039378302     0.0     0.0     0.0
    7   0.0000795287504  -0.2112136303635     0.0     0.0     0.0
    8   0.0000344453712  -0.2112135044391     0.0     0.0     0.0
    9   0.0000191157829  -0.2112134797407     0.0     0.0     0.0
   10   0.0000102110027  -0.2112135739939     0.0     0.0     0.0
 MICROCYCLE DIIS UPDATE:                       10                        5
   11   0.0000014114417  -0.2112137751314     0.0     0.0     0.0
   12   0.0000005469990  -0.2112137029890     0.0     0.0     0.0
   13   0.0000002516295  -0.2112137138349     0.0     0.0     0.0
   14   0.0000001346878  -0.2112137119486     0.0     0.0     0.0
   15   0.0000000715351  -0.2112137115619     0.0     0.0     0.0
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.211213711561925
 CCSD total energy / hartree       =       -76.237998572774032

 Singles contributions

 Doubles contributions

 Parallel integral file used      16 records with       0 large values


 Task  times  cpu:        1.1s     wall:        1.3s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 2059     2059     1.91e+04 2391     4936        0        0     5886     
number of processes/call 1.53e+00 1.63e+00 1.93e+00 0.00e+00 0.00e+00
bytes total:             3.24e+07 6.10e+06 1.79e+06 0.00e+00 0.00e+00 4.71e+04
bytes remote:            1.36e+07 1.20e+06 9.55e+05 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 3551072 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        33
	current total bytes		         0	         0
	maximum total bytes		   1061744	  22509656
	maximum total K-bytes		      1062	     22510
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
    J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata, W. Ma,
       M. Klemm, O. Villa, F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, R. J. Harrison,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
    M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
      G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall,
      J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt,
      P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall,
    D. Elwood, E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson,
     J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        1.1s     wall:        1.4s
